tests/eval_files/async900.py

```diff
@@ -54,3 +54,12 @@ async def cm():
 async def another_non_generator():
     def foo():
         yield
+
+
+# ARG --transform-async-generator-decorators=this_is_like_a_context_manager
```

Comment:
Minor: I'd probably have this at the top of the file for readability (and in all other files I've had them at the top), esp as it pops up in the other error messages.